C01/credit-batch-metadata/C01-20150101-20151231-005.json

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "regen:additionalCertifications": { "@container": "@list" }
+  },
+  "@type": "regen:CreditBatch",

```

Comment:
let's start using this instead: `"@type": "regen:C01-CreditBatch", see template: https://github.com/regen-network/regen-registry-standards/blob/main/jsonld/credit-batches/C01-verified-carbon-standard-batch.json and related convo: https://github.com/regen-network/regen-registry-standards/pull/16#discussion_r874473527

Reply:
@S4mmyb up